src/cpu/flexcpu/inflight_inst.cc

```diff
@@ -276,7 +323,10 @@ InflightInst::readVecRegOperand(const StaticInst* si, int op_idx) const
 TheISA::VecRegContainer&
 InflightInst::getWritableVecRegOperand(const StaticInst* si, int op_idx)
 {
-    // TODO this one may produce bad results, need to rethink this.
+    // TODO this one may produce bad results, need to rethink this, since if
```

Comment:
You could add an assert here and make sure that things match your expectations :)

Reply:
Oh no, that's just something I'm fairly certain I have to change eventually, I was just fleshing out the comment